README.md

```
@@ -38,97 +38,105 @@ AS-StageFX provides a collection of performant, customizable, and audio-reactive
 
 ---
 
-## Available Shaders
+## Shader Statistics
 
-AS-StageFX includes **51 shaders** across four categories: **24 Background (BGX)**, **4 Graphic (GFX)**, **3 Lighting (LFX)**, and **20 Visual (VFX)** effects.
+- **Total Shaders:** 58
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)